solr/packaging/test/test_packages.bats

```diff
@@ -40,9 +40,29 @@ teardown() {
   run solr package
   refute_output --partial "No Solr nodes are running."
 
-  run solr package -help
+  run solr package help
   assert_output --partial "Add a repository to Solr"
 
   run solr package list-available
   assert_output --partial "Available packages:"
 }
+
+@test "deploying and undeploying of packages" {
+  run solr start -c -Denable.packages=true
+
+  run solr package
+  refute_output --partial "No Solr nodes are running."
+
+  run solr package help
+  assert_output --partial "Bootstraps a previously installed package"
+  assert_output --partial "Undeploys a package from specified collection(s)"
+
+  # Deploy package - the package doesn't need to exist before the collection validation kicks in
+  run solr package deploy PACKAGE_NAME -collections foo-1.2
+  # assert_output --partial "Deployment successful"
```

Comment:
I can't get these working on Windows, so I don't know if there'll be any positive output to assert for (I expect not), but this is here in case it turns out there's something usable in the log output when someone is able to get tests running :D

Reply:
okay, taking a looksee now...